ci-operator/step-registry/openshift/manifests/crun/openshift-manifests-crun-commands.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+cat >> "${SHARED_DIR}/manifest_mc-master-crun.yml" << EOF
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-master-crun
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,W2NyaW8ucnVudGltZV0KZGVmYXVsdF9ydW50aW1lID0gImNydW4iCltjcmlvLnJ1bnRpbWUucnVudGltZXMuY3J1bl0KcnVudGltZV9yb290ID0gIi9ydW4vY3J1biIK
+        filesystem: root
+        mode: 0644
+        path: /etc/crio/crio.conf.d/50-crun
+EOF
+
+cat >> "${SHARED_DIR}/manifest_mc-worker-crun.yml" << EOF
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
```

Comment:
nit: might be slightly easier to maintain going foward if you keep the `manifest_mc-master-crun.yml` as you have it, and then:

```sh
sed 's/master/worker/g' "${SHARED_DIR}/manifest_mc-master-crun.yml" > "${SHARED_DIR}/manifest_mc-worker-crun.yml"
```

Or some such to write the control plane manifest.